package.json

```diff
@@ -18,7 +18,7 @@
     "Lluis Agusti <hi@llu.lu>"
   ],
   "dependencies": {
-    "next": "latest",
+    "next": "^13.4.5",
```

Comment:
I recommend to using 'latest' for project with context as example, and let the exact version saved in the lockfile.